pkg/compiler/debug.go

```diff
@@ -110,11 +110,13 @@ type DebugRange struct {
 
 // DebugParam represents the variable's name and type.
 type DebugParam struct {
-	Name         string                  `json:"name"`
-	Type         string                  `json:"type"`
-	RealType     binding.Override        `json:"-"`
-	ExtendedType *binding.ExtendedType   `json:"-"`
-	TypeSC       smartcontract.ParamType `json:"-"`
+	Name string `json:"name"`
+	// Type is a string representation of stackitem.Type of the variable (VM's type).
+	Type         string                `json:"type"`
+	RealType     binding.Override      `json:"-"`
+	ExtendedType *binding.ExtendedType `json:"-"`
+	// TypeSC is a smartcontract.Type of the variable (SmartContract's type).
```

Comment:
No functional changes -> something different.

Reply:
Moved to a separate PR.